Formula/a/adapterremoval.rb

```diff
@@ -0,0 +1,27 @@
+class Adapterremoval < Formula
+  desc "Rapid adapter trimming, identification, and read merging"
+  homepage "https://github.com/MikkelSchubert/adapterremoval"
+  url "https://github.com/MikkelSchubert/adapterremoval/archive/refs/tags/v2.3.4.tar.gz"
+  sha256 "a4433a45b73ead907aede22ed0c7ea6fbc080f6de6ed7bc00f52173dfb309aa1"
+  license "GPL-3.0-or-later"
+
+  uses_from_macos "bzip2"
+  uses_from_macos "zlib"
+
+  def install
+    ENV.deparallelize
```

Comment:
maybe an upstream bug report for it?

Reply:
Upstream is moving to `meson` for building in 3.x, 2.x is still using a homemade makefile.

Reply:
i see, make sense 👍